spec/models/note_spec.rb

```diff
@@ -12,7 +12,7 @@
       it 'with a nil body, return default error message' do
         note = build(:note, body: nil)
         note.save
-        expect(note.errors[:body]).to eq(["Body can't be blank"])
+        expect(note.errors[:body]).to eq(["can't be blank"])
```

Comment:
The integration of the component looks good - I'll check it locally before thumbing.